app/views/users/index.html.erb

```diff
@@ -1 +1 @@
-<h1>Hello users welcome to <%= @title %></h1>
```

Comment:
- [ ] The _show_ action is improperly defined in the _UsersController_. Kindly update the _show_ action in the _UsersController_ to look like:
```ruby
def show
  @user = User.find(params[:id])
end
```

Reply:
This has been fixed. Thanks for picking it up.